.gitignore

```diff
@@ -22,3 +22,5 @@
 /www/images/
 /plugins_repo/build-ap.xml
 /plugins_repo/release/*.zip
+.idea/
+*.iml
```

Comment:
Those kinda belong to your own global git ignore.

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore